js/data/bucket/line_bucket.js

```diff
@@ -159,16 +159,18 @@ class LineBucket extends Bucket {
 
             // Determine the normal of the join extrusion. It is the angle bisector
             // of the segments between the previous line and the next line.
-            let joinNormal = prevNormal.add(nextNormal)._unit();
+            let joinNormal = prevNormal.add(nextNormal);
 
-            if (isNaN(joinNormal.x) && isNaN(joinNormal.y) && join === 'miter') {
+            if (joinNormal.x === 0 && joinNormal.y === 0 && join === 'miter') {
```

Comment:
Thanks @kkaefer — didn't give this the time I should have yesterday :bowing_woman: